lib/foreman_theme_satellite/branded_words.rb

```diff
@@ -53,6 +53,6 @@ module ForemanThemeSatellite
     /\b[Kk]atello 4.11\b(?!-)/       => 'Satellite 6.15',
     /\b[Kk]atello 4.12\b(?!-)/       => 'Satellite 6.16',
     /\b[Kk]atello 4.13\b(?!-)/       => 'Satellite 6.16',
-    %r{https://theforeman.org/plugins/katello/(?!-)} => "https://access.redhat.com/documentation/en-us/red_hat_satellite/#{ForemanThemeSatellite::SATELLITE_SHORT_VERSION}/html/managing_hosts/registering_hosts#registering-a-host-to-satellite-using-the-bootstrap-script"
+    %r{https://theforeman.org/plugins/katello/(?!-)} => "https://access.redhat.com/documentation/en-us/red_hat_satellite/#{ForemanThemeSatellite::SATELLITE_SHORT_VERSION}/html/managing_hosts/registering_hosts_to_server_managing-hosts#Registering_Hosts_by_Using_the_Bootstrap_Script_managing-hosts"
```

Comment:
Has been changed to "docs.redhat.com": https://github.com/theforeman/foreman-documentation/pull/3098 cc @asteflova

Reply:
I wonder where this even came from. I can't figure out where this was used.

Reply:
https://github.com/RedHatSatellite/foreman_theme_satellite/pull/59 at least does the docs part.

Reply:
> I wonder where this even came from. I can't figure out where this was used.

@ekohl : https://bugzilla.redhat.com/show_bug.cgi?id=1599310

Reply:
Subdomain changed from `access` to `docs`.

Reply:
Ah, that was dropped in https://github.com/theforeman/foreman_remote_execution/pull/820 so it's gone for more than 3 years already.